src/lib/bootloader/auto_client.rb

```diff
@@ -27,7 +31,19 @@ def run
     end
 
     def import(data)
-      Yast::Bootloader.Import(data)
+      begin
+        Yast::Bootloader.Import(data)
+      rescue Bootloader::UnsupportedBootloader => e
+        textdomain "bootloader"
+        possible_values = BootloaderFactory.supported_names + [BootloaderFactory::DEFAULT_KEYWORD]
+        AutoInstall.issues_list.add(:invalid_value, "bootloader", "loader_type",
+            e.bootloader_name,
+            _("The selected bootloader is not supported on this architecture. Possible values: ") +
```

Comment:
I just noticed it's a single line string. How is that rendered, is it displayed in a RichText? Will that fit the 80x25 ncurses screen?

Reply:
I have no clue, it is up to autoyast how it will show it.

Reply:
It should fit. There will be line breaks: https://trello.com/c/g5GxOY2c/1416-5-ay-implement-checking-for-firewall-part-of-the-profile